### PR TITLE
improve repo hygiene and add readme

### PR DESCRIPTION
**Pull request by morningvera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,101 @@
-# Polygon Golang Client - WebSocket & RESTful APIs
+# Polygon Go Client
 
-[![Build Status](https://drone.polygon.io/api/badges/polygon-io/client-golang/status.svg)](https://drone.polygon.io/polygon-io/client-golang)
+<!-- todo: add a codecov badge -->
 
-# Coming Soon
\ No newline at end of file
+[![Docs][doc-img]][doc] [![Build][build-img]][build]
```
Comment (morningvera) [inline README.md:5]: the build badge won't show up until we merge to master

File: README.md [hunk 2]
```diff
@@ -1,5 +1,101 @@
...
+
+The official Go client library for the [Polygon](https://polygon.io/) REST and Websocket API.
```
Comment (mcdayoub) [inline README.md:7]: Since we capitalize the s in `WebSocket` in our docs, lets do the same here

File: README.md [hunk 3]
```diff
@@ -1,5 +1,101 @@
...
+
+`go get github.com/polygon-io/client-go`
+
+See the [docs](https://polygon.io/docs/stocks/getting-started) for more details on our API. 
+
+This client is still in pre-release. It only supports the REST API but Websocket support is coming soon. The public interface is relatively stable at this point but is still liable to change slightly until we release v1.
```
Comment (mcdayoub) [inline README.md:13]: Same

**Review by mcdayoub — CHANGES_REQUESTED** [review]
just the WebSocket change

**Review by mcdayoub — APPROVED** [review]

**Pull request merged**